services/auth/test_api.sh

```
@@ -0,0 +1,102 @@
+#remember to install jq for output formatting - apt-get install jq
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)